src/Query/User/PackageQuery/DbalPackageQuery.php

```diff
@@ -103,6 +104,29 @@ public function getById(string $id): Option
         return Option::some($this->hydratePackage($data));
     }
 
+    /**
+     * @return Version[]
+     */
+    public function getVersions(string $packageId): array
```

Comment:
it seems to me that it should be a paged list, because the quantity may grow, but at the moment it can stay that way (we can do it in a separate PR)